tests/bcr/MODULE.bazel

```diff
@@ -30,6 +30,9 @@ go_sdk.host(version = "3.0.0")
 # Bring the default SDK into scope to verify that it exists.
 use_repo(go_sdk, "go_default_sdk")
 
+# Bring the selected host compatible SDK into scope to verify that it exists.
+use_repo(go_sdk, "go_host_compatible_sdk")
```

Comment:
On second thought, could we rename this to `go_host_compatible_sdk_label` or something similar? This is very much a different repo from `go_default_sdk`.